examples/fluids/navierstokes.c

```diff
@@ -197,14 +197,7 @@ int main(int argc, char **argv) {
   // -- Fix multiplicity for ICs
   PetscCall(ICs_FixMultiplicity(dm, ceed_data, user, user->Q_loc, Q, 0.0));
 
-  // ---------------------------------------------------------------------------
-  // Set up lumped mass matrix
-  // ---------------------------------------------------------------------------
-  // -- Set up global mass vector
-  PetscCall(VecDuplicate(Q, &user->M_inv));
-
-  // -- Compute lumped mass matrix
-  PetscCall(ComputeLumpedMassMatrix(ceed, dm, ceed_data, user->M_inv));
+  PetscCall(CreateKspMassOperator(user, ceed_data));
```

Comment:
```suggestion
  PetscCall(CreateKSPMassOperator(user, ceed_data));
```
minor - PETSc capitalizes `KSP` in function names

Reply:
Ah, yep.